### PR TITLE
Kevin/new mode

### PR DESCRIPTION
**Pull request by RocketKevin**
-Reposition objects
-Added Same mode

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ import './App.css';
 
 //Holds one true value
 let valueBoard = [];
-let mode = "easy";
+let difficultyValue = 46;
```
Comment (brotatotes) [inline src/App.js:6]: `numbersToHide` or `numbersToRemove` is probably better.

File: src/App.js [hunk 2]
```diff
@@ -141,17 +140,41 @@ function stateOfWorld() {
...
+
+function medium() {
+  localStorage.setItem("difficultyValue", 30);
   window.location.reload();
 }
 
 function hard() {
-  var mode = "hard";
-  localStorage.setItem("mode", mode);
+  localStorage.setItem("difficultyValue", 46);
```
Comment (brotatotes) [inline src/App.js:153]: if these are hardcoded (which is fine), why do we have the global variable?

Comment (RocketKevin) [reply src/App.js:153]: It make things easier for now.

File: src/App.js (hunk 1 above)
Comment (RocketKevin) [reply src/App.js]: Done

**Pull request merged**